src/modules/RA/DataLayer/components/tabs/StyleTab/Style/ColorListField.js

```diff
@@ -35,13 +40,55 @@ const ColorListField = ({ value, onChange = () => {}, maxClasses = DEFAULT_MAX_C
   return (
     <div className={classes.colorList}>
       {(value || []).map((color, index) => (
-        <ColorPicker
+        <>
```

Comment:
I think a `key` prop must be put here `<React.Fragment key={index}>`
And the `key` line.63 is useless

Reply:
Linter is telling me `Do not use Array index in keyseslintreact/no-array-index-key`

Reply:
I meant l.64, and if you cannot use anything else but `index` you should keep `// eslint-disable-next-line react/no-array-index-key`

Reply:
Done.